src/utils/issue.ts

```diff
@@ -233,10 +233,11 @@ export function getOwnerRepoFromHtmlUrl(url: string) {
 
 export async function getAvailableOpenedPullRequests(context: Context, username: string) {
   const { reviewDelayTolerance } = context.config;
-  if (!reviewDelayTolerance) return [];
+  if (!reviewDelayTolerance) return { approved: [], changes: [] };
 
   const openedPullRequests = await getOpenedPullRequestsForUser(context, username);
```

Comment:
@gentlementlegen I think the reason why I keep getting a 404 is that `openedPullRequests` returns the openIssues for the particular user and not pull request information. When mapping through through `openedPullRequests`. We use `openedPullRequest.number` as the pull request number whereas it's the issue number that's why it returns a `404` for that pull because it doesn't exist. 

When I was testing coincidentally I had an issue opened with the same number as the pull request I used for testing reason why I didnt get the error I guess. 

What's on this branch for `openedPullRequests` is same as whats on development. So I'm not sure how this ever worked to begin with.

Reply:
Then we can add the `is:pr` filter to this line
https://github.com/ubiquity-os-marketplace/command-start-stop/blob/9152118c14d0a9efb4a2b32ee969b62117aab73e/src/utils/issue.ts#L179
Let me know if that resolves your problem.

Reply:
@jordan-ae I just understand now what is going on. When upgrading `octokit` to use the `rest` functions, the behavior differs from `octokit/core` in a sense that it will throw 404 errors if no PR is found. With the core version, it will not throw but return an empty array.

The proper fix is to check for the status code, I'll fix you that on the development branch.

Reply:
Okay I see. With the proposed `is:pr` fix you sent earlier on the seems to solve the issue. Not sure if it's the most optimal solution but with it the feature works correctly. Including the review delay threshold. You can test it now